app/views/companies/show.html.haml

```diff
@@ -49,6 +49,12 @@
     .row.center.item-nav-buttons#item-nav-buttons
 
       - if user_can_edit
+        .dinkurs-fetch
+          = button_to 'get Dinkurs events',
+                      fetch_from_dinkurs_company_events_path(@company),
+                      id: 'getDinkursEvents',
+                      remote: true,
+                      data: {disable_with: "Getting the events..."}
```

Comment:
<img width="862" alt="screen shot 2018-04-05 at 8 00 32 am" src="https://user-images.githubusercontent.com/9968213/38364633-737d5cf2-38a7-11e8-937c-671c0f3cd547.png">

---
^^ Admin view of company `show` view.  

Did you mean to leave some aspects of the initial events UI to a follow-on story?   If not:

1. We need an "events" table - visible to everyone - showing the company event fields (with the sign_up_url causing another window/tab to open).  (the events table should not be present if there are no events)

2. @thesuss can comment on the order in which the tables appear .... I suggest adding the events table after the address(es) table.

3) The "Get Dinkurs Events" button should be
  - styled like the other buttons
  - appear in-line with the other buttons

(later, we may want to redesign this so that the "Add Address" button appears immediately after/near the address table, similarly for the "Get Dinkurs Events" button)

Reply:
views was just copied from: https://github.com/AgileVentures/shf-project/pull/397

Reply:
Really. Hm. I don't have time to install this today but I'd like to see it, Perhaps a view in the start of the meeting?

Reply:
this part: https://github.com/AgileVentures/shf-project/pull/397/files#diff-2674ca98bdddf2829d9e284f8e25e07eR56

Reply:
Ah. Understand. =)